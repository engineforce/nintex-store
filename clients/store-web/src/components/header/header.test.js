import React from 'react'
import renderer from 'react-test-renderer'
import Header from './index'

describe('Header', () => {
  describe('when render header with title', () => {
    const header = renderer.create(<Header siteTitle="Default Starter" />)

    it('renders a link with href to home', () => {
      expect(header.root.findByType('a').props.href).toEqual('/')
    })

    it('renders a link with body that match the title', () => {
      expect(header.root.findByType('a').props.children).toEqual(
        'Default Starter'
      )
    })
  })
})
