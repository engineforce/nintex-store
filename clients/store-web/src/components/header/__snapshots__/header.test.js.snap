// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
<header
  className="sc-bdVaJa llhFdL"
>
  <h1
    className="sc-bwzfXH kabNnl"
  >
    <a
      aria-current="page"
      className=""
      href="/"
      onClick={[Function]}
      onMouseEnter={[Function]}
      style={Object {}}
    >
      Default Starter
    </a>
  </h1>
</header>
`;
